### ISSUE TITLE
Testing!

### ISSUE THREAD

**Issue by sgoodwin** [shrdev/Checklists#9]
Now that we have a roughly functioning app, we need to add some tests so we can change how things work behind the scenes without breaking anything. How much do you know about unit tests and UI tests?

**Comment by shrdev** [comment]
No. I have no knowledge about them. But I will read about them. So, I have to check the functionality by writing unit tests ?!

**Comment by sgoodwin** [comment]
Yes, that's the idea.

Watch these:
https://developer.apple.com/videos/play/wwdc2014/414/
https://developer.apple.com/videos/play/wwdc2015/406/
https://developer.apple.com/videos/play/wwdc2016/409/

And there's this for reference:
https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/04-writing_tests.html

Email me if you have questions about it :)

**Comment by sgoodwin** [comment]
This is the only remaining task for now. Do feel free to email me if you have any questions :) Then we can start writing some tests!
